### PR TITLE
feat(eventing): enable eventing in ha-cluster and ha-node agents

### PR DESCRIPTION
**Pull request by datacore-vvarakantham**
Enables eventing in agent-ha-node and agent-ha-cluster.

## Description

Enables eventing to generate NvmePath events in agent-ha-node and HA events in agent-ha-cluster.

## Regression

No

## How Has This Been Tested?

Manually tested.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.

### CODE REVIEW

**Review by tiagolobocastro — APPROVED** [review]

**Review by chriswldenyer — APPROVED** [review]
LGTM

File: chart/templates/mayastor/agents/core/agent-core-deployment.yaml [hunk 1]
```diff
@@ -90,7 +90,8 @@ spec:
             - "-g=0.0.0.0:50052"
             - "--store=http://{{ .Release.Name }}-etcd:{{ .Values.etcd.service.port }}"
             - "--core-grpc=https://{{ .Release.Name }}-agent-core:50051"{{ if .Values.base.jaeger.enabled }}
-            - "--jaeger={{ .Values.base.jaeger.agent.name }}:{{ .Values.base.jaeger.agent.port }}"{{ end }}
+            - "--jaeger={{ .Values.base.jaeger.agent.name }}:{{ .Values.base.jaeger.agent.port }}"{{ end }}{{ if .Values.eventing.enabled }}
```
Comment (niladrih) [inline chart/templates/mayastor/agents/core/agent-core-deployment.yaml:93]: Should these be put in if jaeger is disabled?

**Review by niladrih — COMMENTED** [review]
Left one comment

**Comment by datacore-vvarakantham** [comment]
bors merge

**Pull request merged**